site/src/components/persona.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Persona
+description: Persona component in the Atlas Design System
+template: standard
+figmaEmbed: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FuVA2amRR71yJZ0GS6RI6zL%2F%25F0%259F%258C%259E-Atlas-Design-Library%3Ftype%3Ddesign%26node-id%3D1284%253A2163%26mode%3Ddesign%26t%3DklysUJ7ALWgcF1SQ-1 allowfullscreen
+classType: Component
+classPrefixes:
+  - persona
+---
+
+# Persona
+
+Persona component comes in handy for user's representation. It consists of two parts - user's avatar and details. Details has their name and optional metadata.
+
+| Class pattern             | Sizes                              | screensize          |
+| ------------------------- | ---------------------------------- | ------------------- |
+| `.persona.persona-<size>` | `xs` `sm` `md (default)` `lg` `xl` | `tablet`, `desktop` |
```

Comment:
how many sizes do we really need?

Reply:
Left only 5 sizes, per the conversation in Figma we can't cut to 3. And removed the responsive ones